rskj-core/src/main/java/co/rsk/validators/TxsMinGasPriceRule.java

```diff
@@ -49,8 +50,9 @@ public boolean isValid(Block block) {
         if(CollectionUtils.isNotEmpty(block.getTransactionsList())) {
             for (Transaction tx : txs) {
                 if (!(tx instanceof RemascTransaction) && tx.getGasPriceAsInteger().compareTo(blockMgp) < 0) {
-                    logger.warn("Tx gas price is under the Min gas Price of the block");
-                    panicProcessor.panic("invalidmingasprice", "Tx gas price is under the Min gas Price of the block");
+                    String logMessage = String.format("Tx gas price is under the Min gas Price of the block tx=%s", Hex.toHexString(tx.getHash()));
+                    logger.warn(logMessage);
+                    panicProcessor.panic("invalidmingasprice", logMessage);
```

Comment:
This panic can be removed

Reply:
I'll remove it in the other PR :sweat_smile: